vllm/sequence.py

```diff
@@ -276,6 +279,9 @@ def lora_int_id(self) -> int:
 
     def get_last_latency(self, now: float) -> float:
         """Gets last token latency for Request level timings."""
+        if self.first_token_time is None:
+            self.first_token_time = now
```

Comment:
It seems the `first_token_time` is only set when `log_stats` is enabled, which is a bit implicit. Is there a better place we can set this field?

Reply:
Yeah, agreed, let me put it somewhere else